Makefile

```diff
@@ -0,0 +1,26 @@
+
+BIN_PREFIX := $(shell npm bin)
+TSC := tsc --module commonjs --target ES5 --noImplicitAny --noEmitOnError --sourceMap
+
+SRCS_TS += $(shell ls *.ts)
+SRCS_JS += $(patsubst %.ts,%.js,$(SRCS_TS))
+
+TSLINT := tslint
```

Comment:
We should use `BIN_PREFIX` to not rely on the user's terminal environment to pick this up.  Take a look at https://github.com/bloomberg/blpapi-http/blob/7f54192b6ebbd6b8e32e93d97606989d66f335c6/Makefile#L34.